### PR TITLE
validate pos L1 duration

### PR DESCRIPTION
**Pull request by arturrez**
## Why this should be merged
adds L1 min staking duration for PoS as 24hrs

### CODE REVIEW

File: pkg/prompts/validations.go [hunk 1]
```diff
@@ -58,6 +58,21 @@ func validateMainnetStakingDuration(input string) error {
...
+	const minL1StakingDuration = 24 * time.Hour
+	d, err := time.ParseDuration(input)
+	if err != nil {
+		return err
+	}
+	if d > genesis.MainnetParams.MaxStakeDuration {
+		return fmt.Errorf("exceeds maximum staking duration of %s", ux.FormatDuration(genesis.MainnetParams.MaxStakeDuration))
+	}
+	if d < minL1StakingDuration {
+		return fmt.Errorf("below the minimum staking duration of %s", ux.FormatDuration(genesis.MainnetParams.MinStakeDuration))
```
Comment (felipemadero) [inline pkg/prompts/validations.go:71]: also fix duration here

File: pkg/prompts/validations.go [hunk 2]
```diff
@@ -58,6 +58,21 @@ func validateMainnetStakingDuration(input string) error {
 	return nil
 }
 
+func validateMainnetL1StakingDuration(input string) error {
+	const minL1StakingDuration = 24 * time.Hour
```
Comment (felipemadero) [inline pkg/prompts/validations.go:62]: ok lets use 2 minutes if possible

File: pkg/prompts/prompts.go [hunk 3]
```diff
@@ -263,6 +264,20 @@ func (*realPrompter) CaptureMainnetDuration(promptStr string) (time.Duration, er
 	return time.ParseDuration(durationStr)
 }
 
+func (*realPrompter) CaptureMainnetL1Duration(promptStr string) (time.Duration, error) {
```
Comment (felipemadero) [inline pkg/prompts/prompts.go:267]: CaptureMainnetL1StakingDuration?

File: cmd/blockchaincmd/add_validator.go [hunk 4]
```diff
@@ -376,7 +376,7 @@ func CallAddValidator(
 			}
 		}
 		if duration == 0 {
-			duration, err = PromptDuration(time.Now(), network)
+			duration, err = PromptDuration(time.Now(), network, pos)
```
Comment (felipemadero) [inline cmd/blockchaincmd/add_validator.go:379]: we know it is pos, maybe clear is using true? as you wish

**Review by felipemadero — APPROVED** [review]
preapproving with comments

File: pkg/prompts/prompts.go (hunk 3 above)
Comment (arturrez) [reply pkg/prompts/prompts.go:267]: agree

File: pkg/prompts/validations.go (hunk 1 above)
Comment (arturrez) [reply pkg/prompts/validations.go:71]: good catch. thank you. fixed.
my bad

**Pull request merged**